app.rb

```diff
@@ -0,0 +1,57 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'database_connection_setup'
+require_relative './lib/peep'
+require_relative './lib/user'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  register Sinatra::Flash
+  set :public_folder, proc { File.join(root, 'static') }
+
+  get '/peeps' do
+    @user = User.find(session[:user_id])
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
```

Comment:
I think your route has been kept skinny, very clean and easy to follow the flow of requests. One question on RESTful route, could you have used parameters in some of the routes?